src/main/java/com/pluralsight/CarDealership/models/Vehicle.java

```diff
@@ -21,14 +23,26 @@ public Vehicle(int vin, int year, String make, String model, VehicleType vehicle
         this.price = price;
     }
 
-    public int getVin() {
+    @Override
+    public String toString(){
```

Comment:
Good use of String.format()

Reply:
As I explained in class you can rearrange the class so the overrides go after the getters/setters and custom methods